resources/settings.xml

```diff
@@ -3,12 +3,17 @@
     <!-- General -->
     <category label="30100">
         <!-- Lutris Executable -->
+        <setting id="custom_path"
+          type="bool"
+          label="30103"
+          default="false" />
+        <!-- Lutris Executable -->
         <setting id="lutris_executable"
             type="file"
             label="30101"
             value="lutris"
-            default="lutris" />
-
+            default="lutris"
+            enable="eq(-1,true)" />
```

Comment:
That is a conditional for the 'path to executable' setting. The option to define the path should only be visible if the user has turned on the use custom path toggle. It sees if the previous setting is true (the toggle) and only shows the option if that is true.